jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -171,6 +171,15 @@ def estimate_tx_size(ins, outs, txtype='p2pkh', outtype=None):
         raise NotImplementedError("Transaction size estimation not" +
                                   "yet implemented for type: " + txtype)
 
+def tx_vsize(tx):
```

Comment:
I like this, because it directly references the underlying library but keeps in the jmbitcoin interface. We should probably fold this into `estimate_tx_size` (in this module) and/or into `jmclient.wallet.estimate_tx_fee` (I guess more the latter). But for now this seems fine.